src/Dock.Avalonia/HostWindowState.cs

```diff
@@ -33,7 +33,7 @@ private void Enter(Point point, DragAction dragAction, IVisual relativeTo)
         {
             var isValid = Validate(point, DockOperation.Fill, dragAction, relativeTo);
 
-            if (isValid == true && _targetDropControl is DockPanel)
+            if (isValid == true && _targetDropControl is Panel)
```

Comment:
We probably need to add custom Panel type here or something otherwise dock target is displayed on tabitem header